apps/nameguard.io/src/app/components/organisms/Header.tsx

```diff
@@ -61,9 +61,9 @@ export const Header = () => {
             >
               Contact
             </Link>
-            <div className="px-5 py-2 rounded-md bg-black text-white hover:bg-gray-800 transition-all duration-300 whitespace-nowrap">
-              <CalButton>Discuss an integration</CalButton>
-            </div>
+            <CalButton className="px-4 py-2 text-sm rounded-md bg-black text-white hover:bg-gray-800 transition-all duration-300 whitespace-nowrap">
+              Discuss an integration
```

Comment:
Can we make this text appear with the same weight as the text of the other buttons / links right next to it? The other  buttons / links right next to it appear to have bold font, while the text in this button doesn't. All these buttons / links should appear bold.